mdb-bi-connector/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+# This should only be used on local
+version: '2'
+services:
+  mdb-bi-connector:
+    build : .
+    networks:
+      - dockernet
+    ports:
+      - 3307:3307
+    command: run.sh
+    environment:
+      MONGO_AUTH: 1
+      MONGO_HOST : "${MONGODB_HOST}"
```

Comment:
🤔 💭 what is this interpolation format doing? is it getting the value from the host environment variable?

Reply:
That is for local, I'll just delete the file. We don't need it. It's to pull in an environment variable.